tools/clang/unittests/HLSL/DxilContainerTest.cpp

```diff
@@ -1762,6 +1763,274 @@ TEST_F(DxilContainerTest, DxcUtils_CreateReflection) {
   }
 }
 
+TEST_F(DxilContainerTest, CheckReflectionQueryInterface) {
+  if (m_ver.SkipDxilVersion(1, 5))
+    return;
+
+  // Check that QueryInterface for shader and library reflection accepts/rejects
+  // interfaces properly
+
+  // Also check that DxilShaderReflection::Get*ParameterDesc methods do not
+  // write out-of-bounds for earlier interface version.
+
+  // Use domain shader to test DxilShaderReflection::Get*ParameterDesc because
+  // it can use all three signatures.
```

Comment:
Patch constants are part of hull shaders. I see the query is successful, but is it returning anything? There's no patch constant function designated in the HLSL source.

Reply:
Patch constants can be read (as input) in the domain shader, and that's all we are compiling.  Since we aren't compiling the hull shader, no patch constant function is used.